README.md

```diff
@@ -65,7 +65,10 @@ require 'virustotal_api'
 file    = '/path/to/file'
 api_key = 'MY_API_KEY'
 
+# for upload file
 vtscan = VirustotalAPI::File.upload(file, api_key)
+# or big file (more than 32MB)
+vtscan = VirustotalAPI::File.upload_big(file, api_key)
```

Comment:
@Grandman can we rename this to: `.upload_large` ?

Reply:
Yes of course